source/known-issues.txt

```diff
@@ -18,4 +16,27 @@ Error List Window on macOS
 Visual Studio for macOS does not have the :guilabel:`Error List` window feature. If
 you are using the {+product+} in Visual Studio for macOS, you will not
 be able to see output from the {+product+}, such as {+query-api+}
-translations, in one location.
\ No newline at end of file
+translations, in one location.
+
+Partial Support for Atlas Search Builders
+-----------------------------------------
+
+The {+product+} provides only partial support for ``Search`` builder
+methods and other Atlas Search features. The {+product+} cannot analyze
+code that uses any of the following features:
+
+- The ``Search.Compound()`` method
+- The ``Search.Range()`` method
+- The ``Search.Equals()`` method with number parameters or parameters of type
+  ``ObjectId`` or ``DateTime``
+- The ``Search.Facet()`` method
+- The ``Search.GeoShape()`` method
+- The ``Search.GeoWithin()`` method
+- The ``Search.MoreLikeThis()`` method
+- The ``Search.Near()`` method with a parameter of type ``DateTime``
+- The ``Search.Wildcard()`` method with multiple field definitions
+- ``Search`` builder methods with parameters of type ``SearchRange``,
+  ``SearchFuzzyOptions``, or ``SearchScoreDefinition``
```

Comment:
I think the *methods* work with the analyzer, but the parameters of these types don't